README.md

```diff
@@ -1,6 +1,6 @@
-# react-router [![Travis][build-badge]][build] [![npm package][npm-badge]][npm]
+# React Router [![Travis][build-badge]][build] [![npm package][npm-badge]][npm]
 
-[`react-router`](https://www.npmjs.com/package/react-router) is a complete routing library for [React](https://facebook.github.io/react).
+React Router is a complete routing library for [React](https://facebook.github.io/react).
```

Comment:
Just a bunch of copy-editing tweaks; I updated repo description to say "A complete routing library for React" as well.